fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/mixin/attachment/ServerPlayerEntityMixin.java

```diff
@@ -0,0 +1,18 @@
+package net.fabricmc.fabric.mixin.attachment;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.server.network.ServerPlayerEntity;
+
+import net.fabricmc.fabric.impl.attachment.AttachmentTargetImpl;
+
+@Mixin(ServerPlayerEntity.class)
+abstract class ServerPlayerEntityMixin implements AttachmentTargetImpl {
+	@Inject(method = "copyFrom", at = @At("TAIL"))
```

Comment:
I will assume that this should be `ServerPlayerEvents.COPY_FROM` so as not to duplicate mixins and simplify FAPI updates for modmuss :D

Reply:
Sure thing.